data/python3-cheat-sheet.json

```diff
@@ -137,6 +137,233 @@
                     "code": "import monmod"
                 }
             ]
-        }
+        },
+        {
```

Comment:
This is wrong. `id` and all other page attributes have already been set. You should only change in items for the previous one.

Reply:
Fixed